index.html

```diff
@@ -6,48 +6,258 @@
+          </ul>
+  
+          <button type="button" class="project-button">
+            See Project
+          </button>
+        </div>
+      </div>
+
+      <div class="project-section project3">
+        <img class="project-img" src="images/SnapshootPortfolio.jpg" alt="">
+  
+        <div class="project-info">
+          <p class="project-title">
+            Tonic
+          </p>
+  
+          <div class="project-info-top">
+            <p class="job-place">
+              CANOPY
+            </p>
+            <img class="circle" src="images/Counter.svg" alt="">
+            <p class="job-title gray">
+              Back End Dev
+            </p>
+            <img class="circle" src="images/Counter.svg" alt="">
+            <p class="project-year gray">
+              2015
+            </p>
+          </div>
+
+          <p class="project-text">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+
+          <ul class="project-languages">
+            <li class="language">html</li>
+            <li class="language">css</li>
+            <li class="language">javaScript</li>
+          </ul>
+  
+          <button type="button" class="project-button">
+            See Project
+          </button>
+        </div>
+      </div>
+
+      <div class="project-section project4">
+        <img src="images/SnapshootPortfolio(3).jpg" alt="">
   
+        <div class="project-info">
+          <p class="project-title">
+            Multi-Post Stories
+          </p>
+  
+          <div class="project-info-top">
+            <p class="job-place">
+              CANOPY
+            </p>
+            <img class="circle" src="images/Counter.svg" alt="">
+            <p class="job-title gray">
+              Back End Dev
+            </p>
+            <img class="circle" src="images/Counter.svg" alt="">
+            <p class="project-year gray">
+              2015
+            </p>
+          </div>
+
+          <p class="project-text">
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+  
+          <ul class="project-languages">
+            <li class="language">html</li>
+            <li class="language">css</li>
+            <li class="language">javaScript</li>
+          </ul>
+
+          <button type="button" class="project-button">
+            See Project
+          </button>
+        </div>
+      </div>
     </section>
 
+    <section id="about-me-section">
+      <h1 class="intro intro-about-me">
+        About <br> myself
+      </h1>
+  
+      <p class="description">
+        Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+  
+      <p class="connect">
+        LET'S CONNECT
+      </p>
```

Comment:
- [ ]OPTIONAL: The subtitle uses a header tag `hx (examples: <h1>,<h2>,<h3>, etc...)`. In this case, it would be nice if you use an `hx` tag instead of `p` tag.

Reply:
@Kandy-Peter Can you please show me where it says that it is a header tag in the figma?  Can you screenshot it maybe? Thanks for the feedback